src/usecases/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const buildIssueUrl = (baseUrl: string, issueKey: string) => {
```

Comment:
**suggestion (non-blocking):** Consider moving this method to a lower layer: e.g., `infrastructure/jira/jira-utils/ts` can be a good fit.